src/components/Vote.jsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { patchVote } from '../utils/api';
+
+
+
+const Vote = ({votes}) => {
+    const [newVotes, setNewVotes] = useState(votes);
+    const[error, setError] = useState(false)
+    const { article_id} = useParams();
+
+    const voteUpdate = (value) => {
+        setNewVotes((currentVotes) => {
+            return currentVotes + value;
+            setError(false)
+        })
+       
+        patchVote(article_id, value).catch(() => {
+             setNewVotes(newVotes);
+             setError(true)
```

Comment:
This is nice that you're catching the error if it fails to patch so a user is informed, I'm not sure about `setNewVotes(newVotes)` because that is setting the state to be the value that it already currently is